spec/database_connection_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'datamapper'
+
+describe DataMapper do
+  describe '.setup' do
+    it 'sets up connection to test db' do
+      expect(DataMapper).to receive(:setup).with(:default, 'postgres://localhost/chitter_test')
+
+      DataMapper.setup(:default, 'postgres://localhost/chitter_test')
```

Comment:
This is not really testing anything and you don't need to test the DataMapper methods.  If you have some other methods that delegate to DataMapper then you could use the same expectation there and a different method call.

If this is only ever used in your Rake task, you can either try to test the Rake task directly or assume that it all works if your other tests pass (and the whole app doesn't "blow up")